### ISSUE TITLE
Feature request: Add button to bookmark am article to the detail page

### ISSUE THREAD

**Issue by avagraphique** [prof18/feed-flow#319]
When reading an article, there's no way to book mark it. Could a button be added to the top, next to the open in browser/share/font size buttons?

Currently to bookmark after reading an article I have to navigate back to the list.

**Comment by prof18** [comment]
You're right, that would be handy to have a bookmark button in the reader mode. 
Thank you for the suggestion, I will put it in the roadmap.
